Update the description of HDS on website to clarify its intended scope (#2012)
hashicorp/design-system: website/docs/about/overview.md

  order: 101
---

## What is a design system

A design system is a set of guidelines, standards, assets, and processes to help organizations design, build, and deploy products rapidly and consistently.

Design systems are often equated with a UI kit or component library. In reality, an effective design system spans the entire lifecycle of a product, from ideation to iteration and everything in between.

## What is Helios

> **Helios** is the design system developed by the HashiCorp Design System Team. It’s commonly referred to as **HDS**.

Reviewer: Not opinionated on this (and there's obviously other context on this page), but using the term 'developed' kind of reads to me like a engineering-focused effort. Is something agnostic better? Built? Created?
```suggestion
**Helios** is the design system built by the HashiCorp Design System Team. It’s commonly referred to as **HDS**.
```

Reviewer 2: Ah, that's a good catch. I like this adjustment.

> Helios is available and recommended for all HashiCorp **product teams using Ember.js**. We also support teams using alternative frameworks or libraries, such as HashiCorp Developer, via our design assets, [tokens](/foundations/tokens), and best practice guidance.

Reviewer 3: The phrasing of this part:

_"We also support teams using alternative frameworks or libraries, **such as HashiCorp Developer**"_

to me kind of sounds like "HashiCorp Developer" itself is an alternative framework/library...

**How about changing to:**
"We also support teams using alternative frameworks or libraries, such as React, via our design assets, [tokens](/foundations/tokens), and best practice guidance."

-OR-

"We also support teams using alternative frameworks or libraries, such as the HashiCorp Developer team which uses React, via our design assets, [tokens](/foundations/tokens), and best practice guidance."

Reviewer 3: See the related suggested edit @Dhaulagiri has now added: https://github.com/hashicorp/design-system/pull/2012#discussion_r1544481682

> Helios is available and recommended for all HashiCorp **product teams using Ember.js**. We also support teams using alternative frameworks or libraries, such as HashiCorp Developer, via our design assets, [tokens](/foundations/tokens), and best practice guidance.

Reviewer 4: ```suggestion
Helios is available and recommended for all HashiCorp product teams using Ember.js. We also support teams using alternative frameworks or libraries, such as React, on a case-by-case basis via our design assets, [tokens](/foundations/tokens), and best practice guidance.
```

Reviewer 3: Nice

> Helios is available and recommended for all HashiCorp **product teams using Ember**. We also support teams using alternative frameworks or libraries, such as HashiCorp Developer, via our design assets, [tokens](/foundations/tokens), and best practice guidance.

Reviewer 4: ```suggestion
Helios is available and recommended for all HashiCorp **product teams using Ember.js**. We also support teams using alternative frameworks or libraries, such as HashiCorp Developer, via our design assets, [tokens](/foundations/tokens), and best practice guidance.
```

Using Helios ensures the different product lines share a common design language. We work closely with these teams on integration and implementation while collecting feedback to understand consumer needs and inform our [roadmap](https://go.hashi.co/hds-rollout).

For more information on how to use the design system, visit our getting started guides for [designers](/getting-started/for-designers) and for [engineers](/getting-started/for-engineers) or visit our public [GitHub](https://github.com/hashicorp/design-system) repository.

## When to use Helios

> ### Consumer-facing products

Reviewer 4: ```suggestion
### Customer-facing products
```

> Helios was specifically created for **consuming-facing products**.

Reviewer 4: ```suggestion
Helios was specifically created for **customer-facing products**.
```

All HashiCorp **product teams** have adopted Helios as the single source of truth for the design language, components, and patterns used to build the UIs of their consumer-facing products.

### Internal products

Helios is **not** meant to be used as a general-purpose UI library.

If you intend to use Helios for **internal products or tools** in HashiCorp, please [contact the Design Systems Team](/about/support) before starting so we can discuss if Helios is a good fit for your team and how this might affect system overhead (changes, maintenance, support, etc).

### External products

Helios is **not** meant to be used outside of HashiCorp. 

> If you’re **not a HashiCorp employee** you’re more than welcome to use the Helios design system as reference, but we don’t provide support if using it to build your own product or UIs. At any moment we may introduce changes to our assets or components that could potentially break your product or UIs.

Author: ```suggestion
If you’re **not a HashiCorp employee** you’re more than welcome to use the Helios design system as reference, but we don’t provide support if you're using it to build your own product or UIs. At any moment we may introduce changes to our assets or components that could potentially break your product or UIs.
```

Reviewer 2: Grammarly suggestions: 
```suggestion
If you’re **not a HashiCorp employee**, you’re more than welcome to use the Helios design system as a reference, but we don’t provide support if you use it to build your product or UIs. At any moment, we may introduce changes to our assets or components that could potentially break your product or UIs.
```
